charts/foundry/templates/deployment.yaml

```diff
@@ -104,7 +104,12 @@ spec:
             - "--balance"
             - {{ .Values.anvil.accountBalance | quote }}
             {{- end }}
-            - "--prune-history"
+            - "--state"
+            - "/state/dump.json"
+            - "--state-interval"
+            - "60"
+            - "--transaction-block-keeper"
```

Comment:
Not sure it's even documented there https://github.com/foundry-rs/foundry/issues/5055 maybe we should put it in `Values.anvil.transaction-block-keeper` since we might tune it in the future?